### ISSUE TITLE
Component - hotgrid - visited contrast

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/adapt-contrib-esdc-college-theme#11]
Page / URL: 4 

The foreground/background colour contrast of the bottom of the number 6 (or top of the number 9) fails the colour contrast tool. People with considerably low vision or colour-blindness may not be able to make out what letter or number this character represents. 

One solution could be to darken the outline of the number so that it’s colour contrast not only passes against the white colour of the number but also against the different colours (ie. purple and light-brown) in the background.

Need to check how that visited color is generated. could it be CSS?
